resources/server/discord.ts

```diff
@@ -37,3 +38,28 @@ export async function reportTweetToDiscord(tweet: Tweet, reportingProfile: Profi
 
   return await axios.post(POST_CHANNEL_URL, { content }, { headers: DISCORD_HEADERS });
 }
+
+export async function reportListingToDiscord(
```

Comment:
Just want to make sure this has been tested, I don't think I've run the twitter version for a few months. Although it should work.

Reply:
Actaully no idea 😂 I'll make sure to test it.

Reply:
Will conduct testing shortly.

Reply:
@erik-sn Issues arose with testing the twitter reporting. I DM'ed you on discord regarding it.

I think it might be best to use a webhook url instead of bot integration as webhooks already use a specified channel and are generally more secure as they don't put the entire server at risk should it get leaked. Would be less convars as well and introduce less failure points when it comes to bug testing/installation instructions. It would still need to be set in the server.cfg the same way to avoid building it though.

I can send you some resources that use it effectively as well if you need reference.

I have no issue writing good documentation for discord bot creation should webhooks not be possible.